tests/conftest.py

```diff
@@ -56,8 +38,30 @@ def tempdir(tmp_path_factory):
 
 @pytest.fixture(scope="session")
 def http_server_url(tempdir, request):
-    url = start_http_server(tempdir, request=request)
-    return url
+    env_port = os.environ.get("PORT", False)
+    port = env_port or get_open_port()
+    host = "127.0.0.1"
+    url = f"http://{host}:{port}"
+    command_list = [
+        "gunicorn",
+        f"--bind={host}:{port}",
+        "--workers=1",
+        "-k",
+        "uvicorn.workers.UvicornWorker",
+        "pangeo_forge_orchestrator.api:api",
+        "--log-level=info",
+    ]
+
+    # the setsid allows us to properly clean up the gunicorn child processes
+    # otherwise those get zombied
+    # https://stackoverflow.com/a/22582602/3266235
+    p = subprocess.Popen(command_list, cwd=tempdir, preexec_fn=os.setsid)
+
+    time.sleep(1)  # let the server start up
+
```

Comment:
How do I run these tests locally?

When I checkout this PR on local and call `pytest -vx` it errors out on a `KeyError` because "DATABASE_URL" is not set.

My first inclination was to add
```python
os.environ["DATABASE_URL"] = url
```
in this fixture... but now I realize that is the API url, not the database URL.

Reply:
How to run the tests locally is described in the README, although perhaps it could clearer:

https://github.com/pangeo-forge/pangeo-forge-orchestrator/pull/27/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R24-R64

You need to set the DATABASE_URL environment variable manually and run the migration before running pytest.

Reply:
Ah-ha. I see that now. Two questions:

1. Does `pwd` in ```export DATABASE_URL=sqlite:///`pwd`/database.sqlite``` mean... process working directory?
2. How do I know what the `pwd` is? Does that need be a... manually-defined temporary directory?

Reply:
`pwd` is a unix command to print the working directory. If you run the command from the project root directory, it will be that directory. You could use any directory you like, as long as it is writeable by you.

Reply:
🤦 Ok, got it! ... can confirm that

```bash
export DATABASE_URL=sqlite:///`pwd`/database.sqlite
python -m alembic upgrade head
pytest -vx
```

works. Now that I know what to look for, I think the README is clear enough for now.